packages/web/src/pages/sign-on/components/desktop/MobileContentContainer.tsx

```diff
@@ -0,0 +1,28 @@
+import { HTMLAttributes, PropsWithChildren } from 'react'
+
+import { Flex, FlexProps } from '@audius/harmony'
+import cn from 'classnames'
+
+import styles from './MobileContentContainer.module.css'
+
+export type MobileContentContainerProps = PropsWithChildren<FlexProps> &
+  HTMLAttributes<HTMLDivElement>
+
+export const MobileContentContainer = ({
+  children,
```

Comment:
might be a bit silly, but i like `const Component => (props: PropType) { ...`with the destructure in body. do you have a preference?